update/state/state.go

```diff
@@ -92,15 +94,18 @@ func FromPbf(filename string, before time.Duration) (*DiffState, error) {
 		timestamp = fstat.ModTime()
 	}
 
-	replicationUrl := "https://planet.openstreetmap.org/replication/minute/"
+	replicationUrl := config.BaseOptions.ReplicationUrl
```

Comment:
To be consistent with the current convention of `before` being passed into the function instead of being read diectly from the config, `replicationUrl` should also be passed in to this function rather than accessed directly from the config. Reading directly from config for some params but not others is confusing.

See [this diff](https://github.com/omniscale/imposm3/pull/164/files#diff-50497b6e28603be277e03d824d9887f3) for an example of where to pass it in to keep things consistent.